Automattic/crowdsignal-forms
# Unwrap all i18n calls

This PR just unwraps all our i18n calls so we use the `@wordpress/i18n` directly.

Once this is merged, next releases should be able to export our translatable strings to the .pot file and get into the wpcom translations flow/cron (ref: p1604357927213600-slack-C02ER2E0Z)

## Test instructions

Checkout and run:

  - `npm run lint:all`
  - `make release`

Verify the strings from JS files are extracted to the .pot file
